dbapi/src/main/java/com/griddynamics/jagger/dbapi/dto/DecisionPerTestGroupDto.java

```diff
@@ -0,0 +1,52 @@
+package com.griddynamics.jagger.dbapi.dto;
+
+import com.griddynamics.jagger.dbapi.entity.DecisionPerTaskEntity;
+import com.griddynamics.jagger.dbapi.entity.TaskData;
+import com.griddynamics.jagger.util.Decision;
+
+import java.util.List;
+
+public class DecisionPerTestGroupDto {
```

Comment:
another option, let's remove test group dto and leave only task dto with list of children task dtos. in this case every test will be a task dto with empty list of children, every test group task dtot with children